.github/workflows/main.yml

```
@@ -73,7 +73,7 @@ jobs:
     - name: Build test bundles for dummy app
       run: cd spec/dummy && rm -rf public/webpack/test && yarn run build:rescript && RAILS_ENV=test NODE_ENV=test bin/${{ matrix.versions == 'oldest' && 'web' || 'shaka' }}packer
     - id: get-sha
-      run: echo "::set-output name=sha::$(git rev-parse HEAD)"
+      run: echo "sha=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:37: Double quote to prevent globbing and word splitting

(shellcheck)